rust/operator-binary/src/airflow_controller.rs

```diff
@@ -723,6 +736,13 @@ fn build_static_envs() -> Vec<EnvVar> {
             value: Some("airflow.api.auth.backend.basic_auth".into()),
             ..Default::default()
         },
+        // The secret key is used to run the webserver flask app and also used to authorize
+        // requests to Celery workers when logs are retrieved.
+        EnvVar {
+            name: "AIRFLOW__WEBSERVER__SECRET_KEY".into(),
```

Comment:
Are the double underscores intentional?
This will get refactored by the comment about the WEBSERVER_SECRET_KEY anyway, but: We should not put the WEBSERVER_SECRET_KEY as clear text into the `env` section of a Pod as this is "public information". Rather we should use a envVar from a Secret. This perfectly aligns with the previous comment on the WEBSERVER_SECRET_KEY

Reply:
As discussed: have replace the unused SECRET_KEY with AIRFLOW__WEBSERVER__SECRET_KEY